kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -306,9 +306,7 @@ class BatchJobSubmission(
         case e: IOException => error(e.getMessage, e)
       }
 
-      MetricsSystem.tracing(_.decCount(
-        MetricRegistry.name(OPERATION_OPEN, statement.toLowerCase(Locale.getDefault))))
```

Comment:
shall we also remove the override of statement in this class ?

Reply:
nice catch, removed

Reply:
also remove the statement override in LaunchEngine: https://github.com/apache/incubator-kyuubi/pull/3077